public/config.example.json

```diff
@@ -2,5 +2,7 @@
   "muzi_root":"https://howl.sdslabs.co.in/",
   "pics_root":"https://music.sdslabs.co.in/muzi_pics/album_thumbnail/",
   "music_root":"https://music.sdslabs.co.in/",
-  "play_root": "https://play.sdslabs.co.in/"
+  "play_root": "https://play.sdslabs.co.in/",
+  "env":"developer" ,
+  "env_OS":"linux"
```

Comment:
Instead of saving the current os version in json, try determining it during runtime using `process.platform`. See http://stackoverflow.com/questions/8683895/how-do-i-determine-the-current-operating-system-with-node-js